### DISCUSSION TITLE
Calendar for Lebach/Germany

### DISCUSSION THREAD

**Discussion by dako5004** [mampfes/hacs_waste_collection_schedule#831] [Q&A]
Hi! Would you include the calendars for Lebach? How can I assist you?

You can find the ics-files here: https://www.lebach.de/lebach/abfall/abfuhrkalender

**Comment by mampfes** [comment]
![image](https://user-images.githubusercontent.com/375950/229294103-1ad70b82-12ed-4505-bf56-5c860a54f541.png)

You can use the link behind "hier: die Ical-Datei zum Importieren auf ihr Mobiltelefon" and configure it as `url` in the [ICS ](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/ics.md)source.

**Reply by dako5004** [reply]
Hi!
Thanks for your quick response. I've already succeeded in importing the ics files. I was just wondering if you could implement them as I installed the update of waste collection schedule on my home assistant today and saw all the new towns that are implemented.
Kind regards,
Daniel

**Reply by 5ila5** [reply]
@dako5004 adding an already supported provider is, if at all, a task with a very low priority. There are still many non -supported providers who do not offer a static URL and are therefore not supported at all.

**Comment by 5ila5** [comment]
officially documented with #865
